database/seeders/DatabaseSeeder.php

```diff
@@ -64,6 +64,7 @@ public function run(): void
             EngagementSeeder::class,
             EngagementResponseSeeder::class,
             DemoEducatableEngagementSeeder::class,
+            SuperAdminSeeder::class,
```

Comment:
Let's add this to the Demo seeder as well for now, so folks on Demo can see this functionality easily.